### ISSUE TITLE
Question about groundtruth pointmap metric for L2W module

### ISSUE THREAD

**Issue by Qmebius** [PKU-VCL-3DV/SLAM3R#6]
Thank you for your work. In your paper, you utilize L2W to align keyframes with the scene frame coordinates. From my understanding, the metric of the registered point map matches that of the initial window point maps, which are used to initialize the scene. The first point map is normalized. Could you clarify how you prepare the ground truth data metrics for training?

**Comment by Ly-kc** [comment]
Hi,

Thank you for your interest in our work! Let me answer your question:  

The training data for the L2W model is composed of a sequence of 12 consecutive images. The first six images serve as scene frames, while the last six are designated as keyframes to be registered (The L2W model supports multi-keyframe co-registration, as detailed in the supplementary materials of our paper).

The pointmaps input to the L2W model from the six scene frames are within a unified coordinate and normalized to a canonical scale together, determined by the average distance of all valid points to the origin. While each pointmap input from the six  keyframes remains in its own local camera coorrdinate system and is normalized individually.

The groud truth output of the L2W model consists of the pointmaps of all twelve frames, sharing the same coordinate system and scale as the six input scene frames.

Hopefully you can find my answer helpful.

**Comment by siyandong** [comment]
closing this issue due to inactivity, please feel free to reopen it if needed
